ui/raidboss/data/03-hw/raid/a12n.txt

```diff
@@ -0,0 +1,140 @@
+161.9 "--sync--" sync /:Alexander:1A18:/ window 161.9,5
+171.6 "Divine Judgment" sync /:Alexander:1AEF:/
+
+# Bridge block to Timegate.
+182.4 "--targetable--"
+185.4 "Chronofoil" sync /:Alexander Prime:19FA:/ window 185.4,5
+199.6 "Temporal Stasis" sync /:Alexander Prime:1AF0:/
+203.7 "--untargetable--"
+203.7 "Plaint Of Solidarity" sync /:Alexander Prime:1AF2:/
+206.9 "--targetable--"
+212.0 "Sacrament" sync /:Alexander Prime:1AE5:/
+217.2 "Divine Spear" sync /:Alexander Prime:1AE3:/
+223.3 "Mega Holy" sync /:Alexander Prime:1AE7:/
+229.6 "Punishing Heat" sync /:Alexander Prime:1AE4:/
+234.8 "Divine Spear" sync /:Alexander Prime:1AE3:/
+244.9 "Temporal Stasis" sync /:Alexander Prime:1AF0:/ window 30,30
+249.0 "--untargetable--"
+249.0 "Plaint Of Solidarity" sync /:Alexander Prime:1AF2:/
+252.2 "--targetable--"
+257.3 "Sacrament" sync /:Alexander Prime:1AE5:/
+266.5 "--sync--" sync /:Alexander Prime:1AE8:/
+268.7 "Blazing Scourge" sync /:Alexander Prime:1AE9:/
+274.6 "Mega Holy" sync /:Alexander Prime:1AE7:/
+278.8 "Divine Spear" sync /:Alexander Prime:1AE3:/
+
+# Timegates seem to push at 50% HP.
+# Push timings during Timegate seem inconsistent, so we include plenty of windows.
+# We don't sync to the 22 log line here because Alexander uses it twice in the bridge block.
+282.8 "--untargetable--"
+286.1 "--targetable--"
+290.1 "Timegate" sync /:Alexander Prime:1A03:/ window 290,5
+309.7 "Divine Spear" sync /:Alexander Prime:1AE3:/
+318.8 "Punishing Heat" sync /:Alexander Prime:1AE4:/
+325.9 "--sync--" sync /:Alexander Prime:1AE8:/
+328.0 "Blazing Scourge" sync /:Alexander Prime:1AE9:/ window 30,5
+332.9 "Sacrament" sync /:Alexander Prime:1AE5:/
+342.1 "Mega Holy" sync /:Alexander Prime:1AE7:/ window 50,5
+346.2 "Divine Spear" sync /:Alexander Prime:1AE3:/
+354.3 "Punishing Heat" sync /:Alexander Prime:1AE4:/
+369.5 "--sync--" sync /:Alexander Prime:1AEA:/
+374.6 "--sync--" sync /:Alexander Prime:1AE8:/
+375.5 "Gravitational Anomaly" sync /:Alexander Prime:1AEB:/ window 100,10
+376.8 "Blazing Scourge" sync /:Alexander Prime:1AE9:/
+378.8 "Divine Spear" sync /:Alexander Prime:1AE3:/
+386.9 "Mega Holy" sync /:Alexander Prime:1AE7:/
+395.0 "Punishing Heat" sync /:Alexander Prime:1AE4:/
+399.2 "Divine Spear" sync /:Alexander Prime:1AE3:/
+408.3 "Temporal Stasis" sync /:Alexander Prime:1AF0:/
+412.4 "Plaint Of Solidarity" sync /:Alexander Prime:1AF2:/
+415.4 "--sync--" sync /:Alexander Prime:1AEA:/
+421.4 "Gravitational Anomaly" sync /:Alexander Prime:1AEB:/
+422.6 "Sacrament" sync /:Alexander Prime:1AE5:/
+429.8 "Incinerating Heat" sync /:Alexander Prime:1AF3:/
+
+# Summon Alexander seems to push at 30% HP.
+# It appears that Alexander has to complete at least a part of the Timegate section first?
+# (Pushes of anywhere from 5% to 30% have been seen.)
+# Technically everything immediately after is a loop.
+# However, there's a fixed enrage after a set number of loops.
```

Comment:
Oh wow, a timeline-based enrage and not a time-based one??

Reply:
Sort of. To be clear, this is where the enrage happens when there are no pushes. I sort of assume that a group that ends up pushing is *probably* not going to see enrage, but it is just possible that if all previous phases are pushed, Alexander would go into a fifth loop, either completing it or breaking at the 12:15 mark to perform the enrage. (Suzaku works this way, for instance, or did before Square fixed the slow/fast instance situation.)

Reply:
Would you mind rolling the loop back up and not unrolling it, then? I think if it technically is an enrage time, then it'd be better to just have the loop run until that gets hit.

I think if the goal is to indicate what the enrage timer is, then maybe there's some way to show this separately than an  entry in the timeline that's not always true.

Reply:
Fair enough, we can do that.